packages/p2p/src/stores/sendbird-store.js

```diff
@@ -123,29 +125,36 @@ export default class SendbirdStore extends BaseStore {
         this.should_show_chat_on_orders = should_show_chat_on_orders;
     }
 
+    setChatError() {
+        this.setHasChatError(true);
+        this.setIsChatLoading(false);
+    }
+
     initialiseChatWsConnection() {
+        this.setFileUploadProperties(null);
+        this.setIsUploadComplete(false);
         this.setHasChatError(false);
         this.setIsChatLoading(true);
 
         const { user_id: sendbird_user_id, token: service_token, app_id } = this.chat_info;
-
         this.sendbird_api = new SendBird({ appId: app_id });
-        this.sendbird_api.connect(sendbird_user_id, service_token, (user, error) => {
-            if (error) {
-                // eslint-disable-next-line no-console
-                console.warn(error);
-                this.setHasChatError(true);
-                this.setIsChatLoading(false);
-            } else {
-                const channel_event_handler = new this.sendbird_api.ChannelHandler();
-
-                channel_event_handler.onMessageReceived = this.onMessageReceived.bind(this);
-                channel_event_handler.onReadReceiptUpdated = this.onReadReceiptUpdated.bind(this);
+        this.sendbird_api
+            .connect(sendbird_user_id, service_token, (user, error) => {
+                if (error) {
+                    // eslint-disable-next-line no-console
```

Comment:
Do we need this console? is there any way to remove this eslint rule?

Reply:
theres quite a few throughout the file, perhaps we can remove these consoles and show a toast error? perhaps this can be a future implementation

Reply:
i think it was previously added just to make the user know what error it was. I agree that showing a toast is perhaps a better way, but for that we might need confirmation from the design team as well. Could be taken up as a future implementation.

Reply:
for now, this is fine, just add a TODO comment and raise this up to the squad channel so they can see what they think about this 🙌